### PR TITLE
Enhancement/seeder

### PR DESCRIPTION
**Pull request by Maximilian118**
Added Seeding.

Eventually, it would be cool to specify what function specifically and amount of iterations in CLI.
As of now though we'll just comment out the unwanted functions when we add more models.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,85 @@
-Setup:
+## Setup:
 
-- npm i
-- npm start
\ No newline at end of file
+1. Install the MongoDB database driver globally: `brew install mongodb` / `npm install mongodb`
```
Comment (samuelbeard) [inline README.md]: Global install should be `npm install -g mongodb`.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,85 @@
...
+
+2. In the root directory of your computer make a directory called data. In that directory make
```
Comment (samuelbeard) [inline README.md]: It's not recommended to put things in the root of a computer. Can this be changed to the user root (`~` not `/`)?

File: README.md (hunk 1 above)
Comment (samuelbeard) [reply README.md]: Also, remove the brew install mongodb. The instructions need to work for linux above anything else.

**Comment by Maximilian118** [comment]
I was on the fence with leaving the tgx file but in the end I landed on... why not?
If anybody screws up the mongo.exec files or whatever all they'd need to do is unpack the new on in the same directory.

File: README.md (hunk 1 above)
Comment (Maximilian118) [reply README.md]: Looking at my globally installed packages I actually don't have a mongoDB driver installed nor do I have a mongoDB driver as a dependency. So I must have done all this without a driver in the first place so I've removed that first setup instruction from the readme.

File: README.md (hunk 2 above)
Comment (Maximilian118) [reply README.md]: It's the MongoDB recommended/default place for the db. I think you can add extra commands to change the db directory yes however this adds unnecessary complication and more room for error.

Comment (samuelbeard) [reply README.md]: I don't have access to the root of my computer as it's a work computer. We can go with this approach if it works on a vagrant box then. As we would always have access to the root.

File: README.md (hunk 1 above)
Comment (samuelbeard) [reply README.md]: Cool. I will test on my computer in a mo.

**Comment by samuelbeard** [comment]
The database is technically not part of the database. The database is a whole other set of software that can be hosted anywhere. 

We also don't want to bloat the repo with easily downloadable stuff.

The tgx file is 80MB and the whole folder contains 317MB. That's HUGE!

File: README.md (hunk 2 above)
Comment (Maximilian118) [reply README.md]: Sounds like a plan.

File: README.md (hunk 1 above)
Comment (Maximilian118) [reply README.md]: Cool

**Comment by Maximilian118** [comment]
Didn't realise it was that big! I've removed it.
The name of the MongoDB folder is enough for people to know what version to download anyway.

**Pull request merged**